components/playlist/browser/resources/components/player.tsx

```diff
@@ -8,23 +8,23 @@ import { bindActionCreators, Dispatch } from 'redux'
 import { connect } from 'react-redux'
 import styled from 'styled-components'
 
-import * as States from 'components/playlist/browser/resources/reducers/states'
+import { ApplicationState } from 'components/playlist/browser/resources/reducers/states'
 import * as playlistActions from '../actions/playlist_action_creators'
-import * as PlaylistMojo from 'gen/brave/components/playlist/common/mojom/playlist.mojom.m.js'
+import { PlaylistItem } from 'gen/brave/components/playlist/common/mojom/playlist.mojom.m.js'
 
 import { getAllActions } from '../api/getAllActions'
 
 export interface Props {
   actions: any
-  currentItem?: PlaylistMojo.PlaylistItem
+  currentItem?: PlaylistItem
   playing?: boolean
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   actions: bindActionCreators(playlistActions, dispatch)
 })
 
-const mapStateToProps = (state: States.ApplicationState) => ({
+const mapStateToProps = (state: ApplicationState) => ({
```

Comment:
Same here. Not actually sure what the current best practices are :laughing:

Reply:
Yeah, player.tsx is likely to be reused so doing it would at this time would be better 👍